documentation/en/supraseal.md

```diff
@@ -68,6 +73,13 @@ LayerNVMEDevices = [
   # Add PCIe addresses for all NVMe devices to use
 ]
 
+# Set to your desiced batch size (what the batch-cpu command says your CPU supports AND what you have nvme space for)
```

Comment:
Are these build-time or run-time flags?

Reply:
Can you give an example of running batch-cpu here?

Reply:
should we simply infer these settings instead of asking for them?

Reply:
Wouldn't hold this PR up on this, opened an issue for tracking https://github.com/filecoin-project/curio/issues/139